src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqTouchstoneRepository.kt

```diff
@@ -10,6 +10,7 @@ import org.vaccineimpact.api.app.repositories.ScenarioRepository
 import org.vaccineimpact.api.app.repositories.SimpleDataSet
```

Comment:
There's a number of changes in this file that I think are just the result of you refactoring and then factoring back again. By all means keep changes that you think are improving readability, but if there are some that are more or less incidental, maybe you revert them so that we get a clearer record of what's actually changed here - and to make it easier to review! I **think** very little has changed here, but I'm not certain.

Reply:
Ah yes, you're right, I'll try revert